src/modifications.jl

```diff
@@ -100,10 +100,26 @@ function modify(
     return throw_modify_not_allowed(ci, change)
 end
 
+function modify(
+    model::ModelLike,
+    cis::Vector{ConstraintIndex},
+    changes::Vector{AbstractFunctionModification},
+)
+    return throw_modify_not_allowed.(cis, changes)
+end
+
 function modify(
     model::ModelLike,
     attr::ObjectiveFunction,
     change::AbstractFunctionModification,
 )
     return throw_modify_not_allowed(attr, change)
 end
+
+function modify(
```

Comment:
It could be like this: https://github.com/jump-dev/MathOptInterface.jl/blob/6b50fa0f9e31ef97a06facaad60af7d0b6ae3d2a/src/variables.jl#L171
```julia
function modify(
    model::ModelLike,
    attr::ObjectiveFunction,
    changes::AbstractVector{<:AbstractFunctionModification},
)
    for change in changes
        modify(model, attr, change)
    end
    return
end
```